tests/langchain/conftest.py

```diff
@@ -37,8 +37,8 @@ def reset_autolog(reset_autolog_state):
 
 
 @pytest.fixture(autouse=True)
-def reset_loaded_model_tracker():
-    _LOADED_MODEL_TRACKER.clear()
+def reset_model_tracker():
+    _MODEL_TRACKER.clear()
```

Comment:
If we need to do this in other flavors, we can move this to `tests/conftest.py`.